README.md

```
@@ -1,35 +1,179 @@
+<<<<<<< HEAD
 # Cortex Linux
+<<<<<<< HEAD
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)